daemons/dss-canary/app.py

```diff
@@ -0,0 +1,43 @@
+import boto3
+import os
+import sys
+import logging
+
+from botocore import errorfactory
+from google.cloud import storage
+import domovoi
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), 'domovoilib'))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+from dss.logging import configure_lambda_logging
+logger = logging.getLogger(__name__)
+configure_lambda_logging()
+
+app = domovoi.Domovoi(configure_logs=False)
+delete_timer = os.environ['DSS_CANARY_TIMER']
+create_timer = str(int(os.environ['DSS_CANARY_TIMER']) + 1)
+canaryKey = 'canary/health'
+
+
+@app.scheduled_function('rate({} minute)'.format(delete_timer))
```

Comment:
Instead of creating a new daemon, consider using the GCP Cloud Schedule, which can inject messages into the dss-gs-event-relay pub/sub topic. This will not require the creation of a canary file: https://cloud.google.com/scheduler/docs/

Reply:
@xbrianh i ended up removing the Canary and just keeping the check to specifically the Event Relay.